<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ass2Widget</class>
 <widget class="QWidget" name="Ass2Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3D Studio</string>
  </property>
  <widget class="QGroupBox" name="group_projection">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>201</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Projection</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>21</y>
      <width>178</width>
      <height>224</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>17</number>
     </property>
     <property name="verticalSpacing">
      <number>4</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <item row="0" column="0" colspan="2">
      <widget class="QRadioButton" name="btn_perspective">
       <property name="text">
        <string>Perspecti&amp;ve</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Field of view</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spin_fov">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>20</number>
       </property>
       <property name="maximum">
        <number>160</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Far</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spin_pefar">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>500</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QRadioButton" name="btn_parallel">
       <property name="text">
        <string>Para&amp;llel</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Top</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QDoubleSpinBox" name="spin_top">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Far</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QSpinBox" name="spin_plfar">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>500</number>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Oblique scale</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QDoubleSpinBox" name="spin_oblscale">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.250000000000000</double>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Oblique angle</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QDoubleSpinBox" name="spin_oblangle">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <double>15.000000000000000</double>
       </property>
       <property name="maximum">
        <double>75.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>15.000000000000000</double>
       </property>
       <property name="value">
        <double>45.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_obj_filename">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>(none)</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QPushButton" name="push_file">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_file">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>OBJ File</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>push_file</tabstop>
  <tabstop>spin_top</tabstop>
  <tabstop>spin_plfar</tabstop>
  <tabstop>spin_oblscale</tabstop>
  <tabstop>spin_oblangle</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btn_parallel</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_top</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_parallel</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_plfar</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_parallel</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_oblscale</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_parallel</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_oblangle</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_perspective</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_fov</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_perspective</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_pefar</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
